src/thor/triplegbuilder.cc

```diff
@@ -326,6 +326,14 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
     }
   }
 
+  // Add tagged names to the edge
+  auto tagged_names_and_types = edgeinfo.GetTaggedNamesAndTypes();
```

Comment:
@dgearhart do we need a controller for this.  Like we do for names.

if (controller.attributes.at(kEdgeNames))

Reply:
+1 For consistency I think it makes sense to add the controller attribute for this.

Reply:
yes